optuna/integration/cma.py

```diff
@@ -226,13 +227,16 @@ def _initialize_x0(search_space):
 
         x0 = {}
         for name, distribution in search_space.items():
-            # TODO(nzw0301) support IntLogUniform
             if isinstance(distribution, UniformDistribution):
                 x0[name] = numpy.mean([distribution.high, distribution.low])
             elif isinstance(distribution, DiscreteUniformDistribution):
                 x0[name] = numpy.mean([distribution.high, distribution.low])
             elif isinstance(distribution, IntUniformDistribution):
                 x0[name] = int(numpy.mean([distribution.high, distribution.low]))
+            elif isinstance(distribution, IntLogUniformDistribution):
+                log_high = math.log(distribution.high)
+                log_low = math.log(distribution.low)
+                x0[name] = math.exp(numpy.mean([log_high, log_low]))
             elif isinstance(distribution, LogUniformDistribution):
```

Comment:
This logic appears to be redundant with the `LogUniformDistribution` case. We can remove its redundancy as follows.

```suggestion
            elif isinstance(distribution, LogUniformDistribution) or isinstance(dist, IntLogUniformDistribution):
```